modules/release-notes-template.adoc

```diff
@@ -14,7 +14,7 @@
 // Versions for the components in New features are here (both for 1.29 and 1.29.1):
 // https://gitlab.cee.redhat.com/serverless/p12n-config/-/blob/release-1.29/config.yaml
 
-{ServerlessProductName} X.YY[.Z] is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+{ServerlessProductName} X.YY[.Z] is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {product-title} are included in the following notes.
```

Comment:
This change renders as `New features, updates, and known issues that pertain to OpenShift Serverless on Red Hat OpenShift Serverless are included in the following notes.` 
I guess it's not correct - repeating "Serverless" twice there.

Reply:
It seems to be related to use of `{product-title}` placeholder. It's the same on public docs as well. 

![Uploading Screenshot 2023-11-02 at 18.37.33.png…]()

Reply:
Perhaps product-title used to be a reference to OCP in the previous structure?

Reply:
`{ocp-product-title}` it's then.

Reply:
@mgencur @dsimansk Thank you Martin and David. The problem is indeed a recent change of the value of the `product-title` attribute. I've fixed it throughout the docs.